### PR TITLE
Initial form of the new event and search event pages.

### PR DESCRIPTION
**Pull request by 38programmer61**
It isn't the last version, yet it gives an idea about the main skeleton.

### CODE REVIEW

**Comment by eye-78** [comment]
My comments before you totally complete this branch and request a core review: 

**Issue 1:**
sport type should be selected from a list. you can pick them from an endpoint. alternatively, it is hardcoded in our code base. see 'src/PermanentComponents/SportNames.js'   
you may use react-select to show the alternatives effectively   
<br/>
**Issue 2:**
the code is not working. please commit working codes. you may delete the uncompleted parts or commit before you start working on them
<br/>
**issue 3:**
this probably is already solved but it's a note for other reviewers as well. only a logged in user can create an event. event creation belongs to the upper part of the conditional rendering at 'App.js line 94'
<br/>
**issue 4:**
There may be a minimum number of players to create an event. One may need at least 2 others to play football
<br/>
**issue 5:**
There may be the duration of the event as well. One may want to play a shorter time while others want to play longer
<br/>
<br/>
These issues are also related to backend. You may talk to backend team. We may also work together to solve these issues, let's contact asap

**Comment by eye-78** [comment]
My comments before you totally complete this branch and request a core review: 

**Issue 1:**
sport type should be selected from a list. you can pick them from an endpoint. alternatively, it is hardcoded in our code base. see 'src/PermanentComponents/SportNames.js'   
you may use react-select to show the alternatives effectively   
<br/>
**Issue 2:**
the code is not working. please commit working codes. you may delete the uncompleted parts or commit before you start working on them
<br/>
**issue 3:**
this probably is already solved but it's a note for other reviewers as well. only a logged in user can create an event. event creation belongs to the upper part of the conditional rendering at 'App.js line 94'
<br/>
**issue 4:**
There may be a minimum number of players to create an event. One may need at least 2 others to play football
<br/>
**issue 4:**

File: front-end/playground/package.json [hunk 1]
```diff
@@ -2,25 +2,32 @@
   "name": "playground",
   "version": "0.1.0",
   "private": true,
-  "proxy": "http://18.217.235.17:8080",
+  "proxy": "http://3.20.232.108:8080",
   "dependencies": {
     "@emotion/react": "^11.6.0",
     "@emotion/styled": "^11.6.0",
+    "@googlemaps/react-wrapper": "^1.1.16",
```
Comment (eye-78) [inline front-end/playground/package.json:9]: We may uninstall this package. Let's talk on it later

**Comment by eye-78** [comment]
I have some more comments after I reviewed your initial codes. Please see them as 'comments' as these codes are not the final product.

Search page will probably work efficiently but it looks a bit old. Results will appear only after submitting the filter form. Here is my suggestion: we may implement a more responsive page. We may store events and filters on two states and would update events when filter states are changed. To prevent excessive re-rendering, we may wait for some time for user to finish their settings. I can continiue from where you left the work or we may work together. See you on slack

**Comment by 38programmer61** [comment]
I have tried to make the corresponding changes. Could you check if it's working?

**Review by eye-78 — APPROVED** [review]
They look good. Thank you for your effort. 

You can use more css so that pages would look nicer. For example, form inputs have labels and they are empty now. And you can set minimum width (in percentage) to the cards so that they would fill the page

I think this commit is useful and we can merge it. I want to use these features in my future branches

**Comment by eye-78** [comment]
@salihozcelik can you review the code so that we can merge it

**Comment by 38programmer61** [comment]
I have tried to complete the functionality part. Styling isn't the last version since it will probably change when we merge all our codes.

File: front-end/playground/src/pages/SearchPage/EventInfo.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#title {
+    color: green;
```
Comment (eye-78) [inline front-end/playground/src/pages/SearchPage/EventInfo.css:2]: Why do you choose green?

**Review by eye-78 — COMMENTED** [review]
Let's merge these changes

**Pull request merged**